db/migrate/20230318000320_add_uid_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUidToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :uid, :string
```

Comment:
what about all the other stuff in `auth_info`?

Reply:
probably going to need something like `@user.identities` - https://auth0.com/docs/manage-users/user-accounts/user-account-linking in order to store refresh tokens per-account